services/credential-issuance-server/src/cli/multiSigIncept.ts

```diff
@@ -0,0 +1,168 @@
+import { SignifyClient, randomPasscode, Tier, ready as signifyReady, Algos, Siger, d, messagize } from "signify-ts";
+import qrcode from "qrcode-terminal";
+import { createInterface } from "node:readline/promises";
+import { stdin, stdout } from 'node:process';
+import { SignifyApi } from "../modules/signify/signifyApi";
+import { waitAndGetDoneOp } from "../modules/signify/utils";
+
+async function getClient(): Promise<SignifyClient> {
+  const client = new SignifyClient(
+    SignifyApi.LOCAL_KERIA_ENDPOINT,
+    randomPasscode(),
+    Tier.low,
+    SignifyApi.LOCAL_KERIA_BOOT_ENDPOINT
+  );
+
+  await client.boot();
+  await client.connect();
+  return client;
+}
+
+// @TODO - foconnor: any.
+async function createIdentifier(client: SignifyClient, name: string): Promise<any> {
+  const createOp = await (await client.identifiers().create(name)).op();
+  await waitAndGetDoneOp(client, createOp);
+  await client.identifiers().addEndRole(
+    name,
+    SignifyApi.DEFAULT_ROLE,
+    client.agent!.pre
+  );
+  return client.identifiers().get(name);
+}
+
+async function waitForFirstNotification(client: SignifyClient, route: string, timeout = 5000, interval = 250) {
+  const startTime = Date.now();
+  while (Date.now() - startTime < timeout) {
+    const notes = (await client.notifications().list()).notes;
+    const note = notes.find((note) => note.a.r === route && note.r === false);
+    if (note) {
+      await client.notifications().mark(note.i);
+      return note.a.d;
+    }
+    await new Promise(resolve => setTimeout(resolve, interval));
+  }
+  throw new Error(`Notification on route ${route} not appearing after ${timeout}ms`);
+}
+
+async function main() {
+  const idwOobi = process.argv[2];
+  if (!idwOobi?.includes("/oobi")) {
+    console.error(`Invalid OOBI argument: ${idwOobi}`);
+    process.exit(1);
+  }
+  
+  await signifyReady();
+  
+  const alice = await getClient();
+  const aliceAid = await createIdentifier(alice, "alice");
+  const oobiA = await alice.oobis().get("alice");
+  console.info(`Alice created: ${aliceAid.prefix}`);
+  
+  const bob = await getClient();
```

Comment:
I think we should rename these to "aliceClient" and "bobClient"

Reply:
done